dev-requirements.txt

```diff
@@ -10,7 +10,7 @@ tox==3.15.2
 twine==3.1.1
 ## The following requirements were added by pip freeze:
 appdirs==1.4.4
-bleach==3.1.5
+bleach==3.3.0
```

Comment:
is there a way to teach snyk about `dev-requirements.txt.in`? (see the comment at the top of this file -- we update the `.in` file instead of this one)